### DISCUSSION TITLE
JustMonikaAfterStory Help with mod_assets in folder C

### DISCUSSION THREAD

**Discussion by KendraStone** [Monika-After-Story/MonikaModDev#9809] [general - spritepacks]
Hi I've found new clothes in my JMAS files that I hadn't seen before but don't know how to gift them to her. It's in the C folder in mod_assets. Do I just drag the file to her character file or save it in a document as a gift then put it there. I have no clue. I'm still new to messing around with game files. Please help.

**Comment by Booplicate** [comment]
Not everything you see there is giftable.

**Reply by KendraStone** [reply]
oh i see. Wonder when she's gonna wear that Nou shirt.

**Reply by Booplicate** [reply]
For that one you need to play NOU and it will eventually get unlocked.

**Reply by KendraStone** [reply]
I've been doing nothing but playing NOU :( I love that game.

**Reply by Booplicate** [reply]
It's also random, so can take some time, if you're actively playing NOU, then you will get it.
